### PR TITLE
deps: bump minimum vscode version to 1.83

### PR DESCRIPTION
**Pull request by Hweinstock**
## Problem
Finishing up: https://github.com/aws/aws-toolkit-vscode/pull/4787

## Solution

---

License: I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by justinmk3** [comment]
the changelog from https://github.com/aws/aws-toolkit-vscode/pull/4787 definitely seems worth having

File: packages/core/src/test/credentials/auth.test.ts [hunk 1]
```diff
@@ -586,9 +585,9 @@ describe('Auth', function () {
         })
 
         it('reauthenticates a connection if the user selects an expired one', async function () {
-            if (isMinVscode('1.83.0')) {
-                this.skip()
-            }
+            // if (isMinVscode('1.83.0')) {
+            //     this.skip()
+            // }
```
Comment (justinmk3) [inline packages/core/src/test/credentials/auth.test.ts:590]: can delete it

File: packages/core/src/test/shared/vscode/runCommand.test.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ import { assertTelemetry, getMetrics, installFakeClock } from '../../testUtil'
 import { getTestWindow } from '../../shared/vscode/window'
 import { makeTemporaryToolkitFolder } from '../../../shared'
 import path from 'path'
-import * as env from '../../../shared/vscode/env'
+// import * as env from '../../../shared/vscode/env'
```
Comment (justinmk3) [inline packages/core/src/test/shared/vscode/runCommand.test.ts:20]: delete?

File: packages/core/src/test/techdebt.test.ts [hunk 3]
```diff
@@ -18,14 +18,10 @@ describe('tech debt', function () {
...
 
         assert.ok(
-            semver.lt(minVscode, '1.75.0'),
-            'remove filesystemUtilities.findFile(), use vscode.workspace.findFiles() instead (after Cloud9 VFS fixes bug)'
-        )
-
-        assert.ok(
-            semver.lt(minVscode, '1.75.0'),
+            semver.lt(minVscode, '1.93.0'),
             'remove AsyncLocalStorage polyfill used in `spans.ts` if Cloud9 is on node 14+'
```
Comment (justinmk3) [inline packages/core/src/test/techdebt.test.ts:25]: I think we actually need the polyfill for web-mode, right @nkomonen-amazon  ?

Comment (nkomonen-amazon) [reply packages/core/src/test/techdebt.test.ts:25]: Yeah I'm pretty sure that is correct. `AsyncLocalStorage` is node specific and I think that in web we are falling back to our polyfill here: https://github.com/aws/aws-toolkit-vscode/blob/d6012a50c6632b8761a866ec23874182c59a1a3c/packages/core/src/shared/telemetry/spans.ts#L29

If we get rid of this then telemetry wont work

File: packages/core/package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
     "license": "Apache-2.0",
     "engines": {
         "npm": "^10.1.0",
-        "vscode": "^1.68.0"
+        "vscode": "^1.83.0"
```
Comment (justinmk3) [inline packages/core/package.json:8]: also in `packages/amazonq/package.json` and `packages/toolkit/package.json`

File: packages/core/src/test/techdebt.test.ts (hunk 3 above)
Comment (justinmk3) [reply packages/core/src/test/techdebt.test.ts:25]: @Hweinstock so this test can be deleted

File: packages/toolkit/.changes/next-release/Removal-cf2f67d7-fd40-4834-960c-bbd57508a7f4.json [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Removal",
+	"description": "Minimum required VSCode version is now 1.83"
```
Comment (justinmk3) [inline packages/toolkit/.changes/next-release/Removal-cf2f67d7-fd40-4834-960c-bbd57508a7f4.json:3]: also need this for amazonq

**Pull request merged**